src/hostname/hostnamed.c

```diff
@@ -430,6 +430,32 @@ static int context_write_data_machine_info(Context *c) {
         return write_env_file_label("/etc/machine-info", l);
 }
 
+// https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/sound/soc/soc-core.c?h=v5.10#n1476
+static const char * const dmi_blocklist[] = {
+        "To be filled by OEM",
+        "TBD by OEM",
+        "Default String",
+        "Board Manufacturer",
+        "Board Vendor Name",
+        "Board Product Name",
+        "N/A",
+        NULL,
+};
+
+static int is_dmi_valid(const char *field)
+{
```

Comment:
coding style: { on same line as function name.

also, booleans should be "bool", not "int", we are not stuck in C89 land